docker/pg_tests/scripts/tests/archiving_ready_rename.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/sh
 set -e -x
-CONFIG_FILE="/tmp/configs/archiving_ready_rename.json"
-COMMON_CONFIG="/tmp/configs/common_config.json"
-TMP_CONFIG="/tmp/configs/tmp_config.json"
-
-cat ${CONFIG_FILE} > ${TMP_CONFIG}
-echo "," >> ${TMP_CONFIG}
-cat ${COMMON_CONFIG} >> ${TMP_CONFIG}
-/tmp/scripts/wrap_config_file.sh ${TMP_CONFIG}
+
+. /tmp/tests/test_functions/prepare_config.sh
+prepare_config "/tmp/configs/archiving_ready_rename.json"
```

Comment:
I was thinking about moving to some `jq` magic - explicitly merge several JSONs.
In this scenario we will achieve 2 goals:
* we use standard tooling
* we have valid JSONs in `docker/pg_tests/scripts/configs`

Unfortunately, quick google-ing about deep JSON merge with `jq` haven't show best solution...